builds/volto-latest.yml

```diff
@@ -0,0 +1,86 @@
+steps:
+
+  # "kaniko" is a build runner whose sole job is to build
+  # container images optimised for use on k8s.
+  # destination is just where on our infrastructure we store
+  # these images.
+  # by default it looks for Dockerfile in the repo root and builds
+  # that though other locations can be specified.
+  # For why we build the image twice with different tags, see
+  # the cicd README.
+  - name: 'gcr.io/kaniko-project/executor:latest'
+    args:
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:latest
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:$SHORT_SHA
+      - context=dir://volto
+      - Dockerfile=Dockerfile-test
+
+  # Builds Volto-test latest container from the artifact registry and Volto tests are run using container volume
+  - name: 'gcr.io/cloud-builders/docker'
+    dir:
+      - '/workspace/volto/'
+    args:
+      - run
+      - -v
+      - /workspace:/workspace
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:latest
```

Comment:
so `--destination` is a kaniko argument for when you're building a new image. I think it's ended up here by accident. this should just be the image path, i.e `${_IMAGE_REPOSITORY}/volto-test:latest`

the syntax `>-` is list, you don't need that either.

Reply:
Removed `--destination` and removed the syntax `>-`